utils/utils.py

```diff
@@ -105,13 +105,45 @@ def get_percentage(error_message):
     return float(error_message.split("values are different ")[1].split("%)")[0][1:])
 
 
-def compare_dataframes(ibis_dfs, pandas_dfs, sort_cols=["id"], drop_cols=["id"]):
+def compare_columns(columns):
+    if len(columns) is not 2:
```

Comment:
```suggestion
    if len(columns) != 2:
```
Don't compare stuff via `is`. You normally need `is` for comparing to a `None` due to ability of classes to override `__eq__` method.

Using `is` in this case would still work, but you'll get unpredictable results if you try to do e.g.
```python
def foo():
    return 10000
def bar(x):
    assert x is 10000
def main():
    bar(foo())
```
this would raise! :)

Reply:
Good suggestion, thanks.